config/database.yml

```diff
@@ -0,0 +1,25 @@
+# SQLite version 3.x
```

Comment:
this file will not be up to GIt, please
`cp database.yml database.yml.example`
`git rm --cached config/database.yml`
and in .gitignore add
`config/database.yml`